README.md

```diff
@@ -45,12 +45,10 @@ Compliance-to-Policy (C2P) bridges Compliance as Code and Policy as Code. C2P ta
 
 Please attend! All are invited.
 
-**When**:
-
-Every other Tuesday starting on April 23, 2024 · 11:00 – 11:30am ET
-[convert to your local time](https://mytime.io/11am/ET)
-
-**Where**: [Zoom](https://zoom-lfx.platform.linuxfoundation.org/meeting/91709345128?password=5510325d-895f-4932-a843-df728dc3028d)
+|          | When    | Where |
+| -------- | ------- | -------- |
+| **Global**  | Every other Tuesday starting on April 23, 2024 · 11:00 – 11:30am ET [convert to your local time.](https://mytime.io/11am/ET)   | [Zoom](https://zoom-lfx.platform.linuxfoundation.org/meeting/91709345128?password=5510325d-895f-4932-a843-df728dc3028d) |
+| **APAC friendly** | Every other following Thursday starting on November 30, 2024 - 05:00 GMT / 16:00 AEDT / 1030 IST [convert to your local time.](https://mytime.io/5am/GMT)    |[Zoom](https://zoom-lfx.platform.linuxfoundation.org/meeting/97945872533?password=abad1bfe-cc9a-49d8-9aa6-99bb469a434d) |
 
 
 **What**: Meeting agenda and notes [Google Docs](https://docs.google.com/document/d/1XTYM7xnWlIqd-8Nn5-qtgvgk8kH3NSmYle5yZvaS7qs/edit?usp=sharing)
```

Comment:
Removes the `What` because the additional makes less sense with this change.

```suggestion
Meeting agenda and notes [Google Docs](https://docs.google.com/document/d/1XTYM7xnWlIqd-8Nn5-qtgvgk8kH3NSmYle5yZvaS7qs/edit?usp=sharing)
```